forge/routes/api/device.js

```diff
@@ -259,10 +259,12 @@ module.exports = async function (app) {
             let changed = false
             if (request.body.name !== undefined && request.body.name !== request.device.name) {
                 request.device.name = request.body.name
+                sendDeviceUpdate = true
                 changed = true
             }
             if (request.body.type !== undefined && request.body.type !== request.device.type) {
                 request.device.type = request.body.type
+                sendDeviceUpdate = true
```

Comment:
I included one extra EV - FF_DEVICE_TYPE since i suspect should a user change this field it is because of a HW change and thus very useful to know inside your flow what HW type is specified or expected (for things like alternate GPIO numbering etc)